docs/development/buildenvironments.rst

```diff
@@ -16,20 +16,19 @@ any development on the Docker build system, you will need to set up `Docker`_ on
 your host system. Setup of Docker will vary by system, and so is out of the
 scope of this documentation.
 
-Once you have Docker set up, you will need to create the base image used for
-container creation. The ``base`` image is found in our `container images repo`_.
-It is the basic container image supported by our community site.
+Once you have Docker set up, you will need to pull down our build image. These
+images are found on our `Docker Hub repository`_, the source comes from our
+`container image repo`_.
 
-To get started, create the image using the `docker` command line tool. You can
-name the image whatever you like here, ``rtfd-build`` is the default name, but
-can be configured in your settings -- see `Configuration`_::
+To get started, pull one of the build images down::
 
-    docker build -t rtfd-build base/
+    docker pull readthedocs/build:latest
```

Comment:
What is the build that it's used in production? I suppose it's the `2.0`, right? If so, I think it's a good idea to mention that `latest` is good for development but to reproduce some someone else's issues it's useful to setup the `2.0`. On the other hand, if `2.0` will be in production for some more time, it's a good idea to test with `2.0`.

Anyway, I'm not sure if this is the proper place to clarify this, maybe another section to this page could be added, or in testing. Not sure.